internal/benchrunner/runners/pipeline/benchmark.go

```diff
@@ -0,0 +1,292 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package pipeline
+
+import (
+	"encoding/json"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/benchrunner"
+	"github.com/elastic/elastic-package/internal/elasticsearch/ingest"
+)
+
+const (
+	// How many top processors to return.
+	numTopProcs = 10
```

Comment:
Should this be configurable? (Can be done in a follow up)

Reply:
This could make sense as a cli flag, can add it in this PR 👍